### DISCUSSION TITLE
“Loop” can’t be opened because Apple cannot check it for malicious software.

### DISCUSSION THREAD

**Discussion by zywang0701** [mrkai77/Loop#337] [Q&A] [answered]
### Bug Description

I manual install Loop from Github. I unzip the file, click the app, with the message popping up, saying that

“Loop” can’t be opened because Apple cannot check it for malicious software.

### Steps To Reproduce

1. Manual Install 
2. Click the app
3. Message pops

### Expected Behavior

I expect it should be installed and run successfully.

### Actual Behavior

Cannot run Loop

### Screenshots

<img width="258" alt="Screenshot 2024-04-20 at 9 21 49 AM" src="https://github.com/MrKai77/Loop/assets/41748102/efb6eba3-02c1-464b-bb64-d95ee61d9e5e">

### MacOS Version

Sonoma 14.3

### Loop Version

1.00 beta.15

### Final Checks

- [X] My issue title is descriptive
- [X] This is a single bug (multiple bugs should be reported individually)

**Comment by mrkai77** [comment, marked as answer]
Yes, you will need to right click `Loop.app` in Finder and click `open`. This is because the app hasn't been notarized by Apple. I plan to do it in the future, but that requires an Apple Developer Program account which I do not yet have.

**Reply by worldofgeese** [reply]
This didn't work for me. I needed to run `xattr -rd com.apple.quarantine /Applications/Loop.app`.
